plugin/handlergen.go

```diff
@@ -250,6 +217,23 @@ func (p *OrmPlugin) generateStrictUpdateHandler(message *generator.Descriptor) {
 	p.P()
 }
 
+func (p *OrmPlugin) generatePreloading(ormable *OrmableType) {
+	var assocList []string
+	for _, fieldName := range p.getSortedFieldNames(ormable.Fields) {
+		field := ormable.Fields[fieldName]
+		if field.GetAssociation() != nil && field.GetHasMany().GetPositionField() == "" {
+			assocList = append(assocList, fieldName)
+		}
+	}
+	if len(assocList) != 0 {
+		preload := ""
+		for _, assoc := range assocList {
+			preload += fmt.Sprintf(`.Preload("%s")`, assoc)
```

Comment:
Yes. This may be needed if association key is not unique across the entire database. I considered only the case when association key is the primary key and therefore unique.

Reply:
Will not need this if we define `AccountID` to be a part of association key.

Reply:
Ah. Right again. Either the FK is unique (account_id unnecessary) or the FK includes the account_id, which should be handled by GORM if done properly.
How about increasing the depth of preload to two though?

Reply:
I don't think it's a good default mainly because that will work poorly for at least contacts-app: you'll got a list of groups containing list of contacts containing list of groups, or list of profiles containing lists of contacts containing profiles.
On the other hand I think we need to made this easily customisable by either
- exposing a hook to customize db query.
- adding an option with a list of associations to preload.
- something different I didn't think about yet.